### ISSUE TITLE
Gath.io is down?

### ISSUE THREAD

**Issue by AsbjornOlling** [lowercasename/gathio#68]
Hi - I can't look up the gath.io domain name.

Recent commits lead me to believe that the project isn't abandoned.
Maybe it's an expired domain name or something?

**Comment by lowercasename** [comment]
Hi there, I can still open gath.io fine and I haven't updated the DNS, so maybe this was a brief server hiccup? Do reopen the issue if you're still getting problems.

**Comment by AsbjornOlling** [comment]
Hi! Thanks for getting back so quickly.

I'm beginning to suspect that it's some kind of issue with a DNS provider *somewhere*. Maybe something pretty far upstream?

It's definitely still down for me. Can't open in browsers, can't query with dig, and dnsdumpster.com also fails to resolve it.
No matter what DNS server I choose (I tried 4, including google and cloudflare) - I get SERVFAIL. More specifically: `EDE: 7 (Signature Expired): (DNSKEY gath.io. signatures, id = 16851)` Some kind of DNSSEC error?

dnssec-analyzer.com also agrees with this error: https://dnssec-analyzer.verisignlabs.com/gath.io

I ran into the error when I tried showing gath.io to a person in Australia (halfway around the globe from me) - and they couldn't resolve the domain either. So it's seemingly not a local thing.

*However* I am able to query it from a server in Frankfurt. :thinking: 
So it's not something totally global either.

I *can* query other domains - including other .io domains, and domains I have never visited before (making cached results unlikely).

Not going to re-open this issue, since I think it's not likely your error- rather an error with some infrastructure provider.

Gonna go wait it out now. My guess is that it's either going to be fixed tomorrow, or propagated around so everybody has the problem tomorrow :rofl:
